easybuild/easyblocks/f/fsl.py

```diff
@@ -63,22 +65,47 @@ def configure_step(self):
         fslmachtype = out.strip()
         self.log.debug("FSL machine type: %s" % fslmachtype)
 
-        # prepare config
-        # either using matching config, or copy closest match
+        # Patch files for ver. < 5.0.10 patch multiple config directories
+        if LooseVersion(self.version) >= LooseVersion('5.0.10'):
+            # Check if a specific machine type directory is patched
+            systype_regex = re.compile("^diff.*config\/(.*(apple|gnu|i686|linux|spark)(?:(?!\/).)*)", re.M)
+
+            patched_cfgs = []
+        
+            for patch in self.patches:
+                patchfile = read_file(patch['path'])
+                res = systype_regex.findall(patchfile)
+                patched_cfgs.extend([i[0] for i in res])
+
+            # Check that at least one config has been found
+            if len(patched_cfgs) != 0:
+                # Check that a single config has been patched
+                if patched_cfgs[1:] == patched_cfgs[:-1]:
```

Comment:
@vanzod This hurts my brain... How about using `nub` from `vsc.utils.missing` and change this to:

```python
if len(nub(patched_cfgs)) == 1:
```

Reply:
I like that. I did not know about `nub`